components/ui/ProjectUpdateNotification.vue

```diff
@@ -0,0 +1,268 @@
+    <div class="more">
+      <button
+        class="iconified-button"
+        @click="toggleExpand()"
+        v-if="versions.length > maxVersions"
+      >
+        <span v-show="!isExpanded"
+          >Show {{ versions.length - maxVersions }} more</span
+        >
+        <span v-show="isExpanded">Collapse</span>
+      </button>
+    </div>
+  </div>
+</template>
+<script>
+import ClearIcon from '~/assets/images/utils/clear.svg?inline'
+import UpdateIcon from '~/assets/images/utils/updated.svg?inline'
+
+export default {
+  name: 'ProjectUpdateNotification',
+  props: {
+    projectTitle: {
+      required: true,
+      type: String,
+    },
+    versions: {
+      required: true,
+      type: Array,
+    },
+    maxVersions: { type: Number, default: 3 },
+  },
+  components: {
+    ClearIcon,
+    UpdateIcon,
+  },
+  computed: {
+    projectPageLink() {
+      return this.versions[0].link.match(/(.*\/.*)\/version\/.*/m)[1] || ''
+    },
+  },
+  methods: {
+    toggleExpand() {
+      this.isExpanded = !this.isExpanded
+    },
+    async clearNotificationGroup(projectName) {
+      await this.$props.versions.forEach(async (el) => {
+        await this.performAction(el, null)
+      })
+    },
+    async performAction(notification, notificationIndex, actionIndex) {
+      this.$nuxt.$loading.start()
+      try {
+        await this.$axios.delete(
+          `notification/${notification.id}`,
+          this.$defaultHeaders()
+        )
+
+        await this.$store.dispatch('user/deleteNotification', notification.id)
+
+        if (actionIndex !== null) {
+          const config = {
+            method:
+              notification.actions[actionIndex].action_route[0].toLowerCase(),
+            url: `${notification.actions[actionIndex].action_route[1]}`,
+            headers: {
+              Authorization: this.$auth.token,
+            },
+          }
+          await this.$axios(config)
+        }
+      } catch (err) {
+        this.$notify({
+          group: 'main',
+          title: 'An error occurred',
+          text: err.response.data.description,
+          type: 'error',
+        })
+      }
+      this.$nuxt.$loading.finish()
+    },
+  },
+
+  data() {
+    return { isExpanded: false }
+  },
+}
+</script>
+
+<style scoped lang="scss">
+.icon {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  svg {
+    height: calc(3rem - var(--spacing-card-sm));
+    width: auto;
+    margin-right: 1rem;
+  }
+}
+.version {
+  display: flex;
+  &:not(:nth-last-of-type(2)) {
+    margin-bottom: var(--spacing-card-sm);
+  }
+
+  :not(.buttons) {
+    // margin-bottom: var(--spacing-card-sm);
+    // display: flex;
+    flex-grow: 1;
+    justify-content: space-between;
+    span {
+      color: var(--color-text-secondary);
+      float: left;
+      margin-right: 1rem;
+    }
+    a {
+      text-decoration: underline;
+      font-weight: bold;
+    }
+  }
+}
+.more {
+  display: flex;
+  justify-content: end;
```

Comment:
Just worth noting that there is this one error when compiling:
```
 WARN  in ./components/ui/ProjectUpdateNotification.vue?vue&type=style&index=0&id=a13cc1a6&scoped=true&lang=scss&                                                                       friendly-errors 16:05:24

Module Warning (from ./node_modules/postcss-loader/src/index.js):                                                                                                                       friendly-errors 16:05:24
Warning

(34:3) end value has mixed support, consider using flex-end instead
                                                                                                                                                                                        friendly-errors 16:05:24
 @ ./node_modules/vue-style-loader??ref--7-oneOf-1-0!./node_modules/css-loader/dist/cjs.js??ref--7-oneOf-1-1!./node_modules/vue-loader/lib/loaders/stylePostLoader.js!./node_modules/postcss-loader/src??ref--7-oneOf-1-2!./node_modules/sass-loader/dist/cjs.js??ref--7-oneOf-1-3!./node_modules/vue-loader/lib??vue-loader-options!./components/ui/ProjectUpdateNotification.vue?vue&type=style&index=0&id=a13cc1a6&scoped=true&lang=scss& 4:14-426 15:3-20:5 16:22-434
 @ ./components/ui/ProjectUpdateNotification.vue?vue&type=style&index=0&id=a13cc1a6&scoped=true&lang=scss&
 @ ./components/ui/ProjectUpdateNotification.vue
 @ ./node_modules/babel-loader/lib??ref--2-0!./node_modules/vue-loader/lib??vue-loader-options!./pages/notifications.vue?vue&type=script&lang=js&
 @ ./pages/notifications.vue?vue&type=script&lang=js&
 @ ./pages/notifications.vue
 @ ./.nuxt/router.js
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi ./node_modules/eventsource-polyfill/dist/browserify-eventsource.js (webpack)-hot-middleware/client.js?reload=true&timeout=30000&ansiColors=&overlayStyles=&path=%2F__webpack_hmr%2Fclient&name=client ./.nuxt/client.js
```
If this could be fixed (or suppressed), that would be appreciated!

Reply:
Done, replaced with `margin-left: auto` (`flex-end` also isn't supported in safari according to mdn)